### DISCUSSION TITLE
Question: What plugin can be used for repeated lotteries?

### DISCUSSION THREAD

**Discussion by chanansh** [jspsych/jsPsych#221] [Q&A]
I am looking for a plugin which can display two buttons and show reward (numerical) according to the choice. Something like the paradigm in Hertwig, R., Barron, G., Weber, E. U., & Erev, I. (2004). 
Essentially the buttons will display option A and option B and will display a reward x once there is a press. x is a random variable.

Hertwig, R., Barron, G., Weber, E. U., & Erev, I. (2004). Decisions from experience and the effect of rare events in risky choice. Psychological Science, 15(8), 534-539.‏

**Comment by jodeleeuw** [comment]
You can do this with the [button-response plugin](http://docs.jspsych.org/plugins/jspsych-button-response/) for getting the user input, and then you will need to use a [function parameter](http://docs.jspsych.org/features/functions-as-parameters/) to show the reward amount using the text or single-stim plugin.

The alternative approach would be to create a modified version of the button-response plugin that builds the response display in.
